src/main/java/mods/eln/Eln.java

```diff
@@ -6689,15 +6689,15 @@ void recipeFuelBurnerItem() {
 		addRecipe(findItemStack("Medium Fuel Burner"),
 			"   ",
 			" Cc",
-			"   ",
-			Character.valueOf('C'), findItemStack("Combustion Chamber", 2),
+			" C ",
```

Comment:
The 3D model of the burner is rather high than long, so I thought this recipe matches better the actual appearance of the model... But I can change it before merging. What do you think?